components/contribution-flow/index.js

```diff
@@ -140,8 +141,9 @@ class ContributionFlow extends React.Component {
       stepDetails: {
         quantity: 1,
         interval: props.fixedInterval || getDefaultInterval(props.tier),
-        amount: props.fixedAmount || getDefaultTierAmount(props.tier),
+        amount: props.paymentMethod === 'crypto' ? 5 : props.fixedAmount || getDefaultTierAmount(props.tier),
```

Comment:
The magical 5 is still there?

Reply:
I've read that this might be removed in another PR. Why not this one?

Reply:
I thought since more updated code are is there we could do it there and this PR is getting a bit long. Maybe I should have put it here. But if it's okay we can revisit this there after this is merged, else I'll do a cherry pick?

Reply:
On second thought I've added it to this PR; now it's all good I think. 👍🏽